zuul.d/kuttl.yaml

```diff
@@ -23,9 +27,11 @@
       - ^zuul.d/kuttl.yaml
     vars:
       cifmw_kuttl_tests_operator_list:
-        - cinder
+        - keystone
```

Comment:
You may ask why, but Cinder kuttl tests aren't so stable atm: https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl